usaspending_api/search/delta_models/award_search.py

```diff
@@ -167,8 +169,24 @@
   awards.latest_transaction_id AS latest_transaction_search_id,
   awards.earliest_transaction_id AS earliest_transaction_search_id,
   awards.category,
-  awards.type,
-  awards.type_description,
+  awards.type AS type_raw,
```

Comment:
I think you left out `type` here. Also make sure that the column order matches between the delta model, sql string definition, and django model.

Reply:
`type` and `type_description` are on lines 174 and 182, respectively. They're the two new `CASE` statements.